summary/summary.py

```diff
@@ -326,4 +298,14 @@ def init(options, configuration, plugin):
     'USD $',
     'What prefix to use for currency'
 )
+plugin.add_option(
+    'summary-availability-interval',
+    300,
+    'How often in seconds the availability should be calculated. Causes RPC and CPU.'
```

Comment:
The meaning of `Causes RPC and CPU` isn't totally clear to me. May be clearer as "uses" or "causes usage"?

Reply:
If your okay, I simply removed that silly addition. I thought it would be useful to tell the user that this is just local RPC and doesn't consume much resources or network stuff. But it doesn't add anything particular interesting.

Solved, rebased an force-pushed.